### PR TITLE
made a change - testing w/ Jon & Michael

### PR DESCRIPTION
**Pull request by trentwirth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # AR_GroundPlane
 Augmented Reality Ground Plane combining Unity with Qualisys
-
+A CHANGE!
```
Comment (jonmatthis) [inline README.md]: hey i don't LIKE this change!

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
 ## Installation & Setup
```
Comment (jonmatthis) [inline README.md:4]: but this is wonderful :D

**Review by jonmatthis — CHANGES_REQUESTED** [review]
change all the changes

File: README.md (hunk 1 above)
Comment (trentwirth) [reply README.md]: we should delete it then D:

Comment (jonmatthis) [reply README.md]: no now I like it

Comment (jonmatthis) [reply README.md]: ok i am satisfgied

**Review by jonmatthis — APPROVED** [review]
wow it great

**Pull request merged**